test/functional/test_framework/wallet.py

```diff
@@ -182,6 +183,26 @@ def sendrawtransaction(self, *, from_node, tx_hex):
         from_node.sendrawtransaction(tx_hex)
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
 
+    def send_large_transactions(
+        self, node, array_of_large_tx, no_of_tx_ids, fee_rate
+    ):
+        # Create large transactions by appending txouts in vout
+        no_of_tx_created = 0
+        for _ in range(no_of_tx_ids):
+            # Create a self transfer here to get the tx details and then append the vout to increase the tx size
+            hex = self.create_self_transfer(from_node=node, fee_rate=fee_rate)['hex']
+            # Converts it into a CTransaction() instance to append the vouts
+            tx_instance = from_hex(CTransaction(), hex)
+            for txout in array_of_large_tx:
+                tx_instance.vout.append(txout)
+            tx_hex = tx_instance.serialize().hex()
+            # Serializes and sends the tx to the nodes
```

Comment:
did you mean this comment to be a line prior? since the serialization is the previous line & the sending is the next line. 

but you could just get rid of this comment. generally we want to add comments if we can explain _why_ we're doing something. and let the code explain _what_ is happening. so in this case, this comment isn't adding any new information.